tests/functional/_intern.js

```diff
@@ -0,0 +1,68 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
(silly question, any reason this file is named `_intern.js`, rather than `intern.js`? Just leftover from renaming everything during the migration?)

Reply:
Yeah it's leftover from renaming, but I thought it was nice that with the underscore it is always the first file listed and has some visual separation from tests files, so I left it like this. But it's nothing important, haha, we can change it back if needed.

Reply:
Ah cool. I like your reasoning -- thanks!